### PR TITLE
Create osrb.md

### PR DESCRIPTION
**Pull request by BrittanyIstenes**
Open Source Review Board - starting point. 

If you have edits or want to make additions, please do so right in the doc to avoid slow downs in publishing and to avoid back and forth

### CODE REVIEW

File: docs/bok/Artifacts/osrb.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+## Creating an Open Source Review Board
+
+Creating an Open Source Review Board (sometimes known as "Advisory Councils) for any open source contributions coming outside of a FINTECH enterprise involves establishing a structured group responsible for evaluating and guiding open source contributions made by the company's employees. This board ensures that these contributions align with the company's strategic interests, comply with legal and regulatory standards, risk and security standards as well as foster positive relationships within the open source community.
```
Comment (robmoffat) [inline docs/bok/Artifacts/osrb.md:3]:
```suggestion
Creating an Open Source Review Board (sometimes known as "Advisory Councils") for any open source contributions coming outside of a FINTECH enterprise involves establishing a structured group responsible for evaluating and guiding open source contributions made by the company's employees. This board ensures that these contributions align with the company's strategic interests, comply with legal and regulatory standards, risk and security standards as well as foster positive relationships within the open source community. 
```

File: docs/bok/Artifacts/osrb.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+## Creating an Open Source Review Board
+
+Creating an Open Source Review Board (sometimes known as "Advisory Councils) for any open source contributions coming outside of a FINTECH enterprise involves establishing a structured group responsible for evaluating and guiding open source contributions made by the company's employees. This board ensures that these contributions align with the company's strategic interests, comply with legal and regulatory standards, risk and security standards as well as foster positive relationships within the open source community. 
+
+**Read below for a structured approach to forming such a board**
+
+### Define the Purpose and Scope
+
+**Objective**
+- Clarify the review board's primary goal, which could include ensuring that open source contributions enhance the company's reputation, comply with legal standards, and align with its strategic goals.
```
Comment (robmoffat) [inline docs/bok/Artifacts/osrb.md:10]: Isn't the purpose of the OSRB first to approve open source _use_.  ie..

**Reviewing and Approving Open Source Usage:** Before teams within an organization incorporate open-source components into their projects, the OSRB reviews these components for any potential legal, security, or compliance issues. This ensures that the organization respects the terms of open-source licenses and avoids potential legal complications.

File: docs/bok/Artifacts/osrb.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...

+
+**Scope**
+- Determine the types of projects or contributions that require review. This might encompass code contributions/enhancements, documentation, participation in open source project governance, starting new open source projects or taking these open source projects into foundation incubation.
+
+### Establish Governance and Policies

+
+**Composition** 
+- The board should include members from diverse backgrounds, such as legal experts specializing in intellectual property and open source licensing, software engineers with experience in open source projects, security experts, risk and representatives from product and strategy teams.
```
Comment (robmoffat) [inline docs/bok/Artifacts/osrb.md:18]: might be best to provide a bullet-list here that people can check off.  Does cover the bases though

File: docs/bok/Artifacts/osrb.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...

+
+**Policies** 
+- Develop clear policies covering aspects such as intellectual property management, contribution guidelines, compliance with open source licenses, security protocols, and conflict of interest disclosures.
```
Comment (robmoffat) [inline docs/bok/Artifacts/osrb.md:21]: Is there a difference between the OSRB and the OSPO?  Seems like with this definition they overlap quite a lot.

Comment (robmoffat) [reply docs/bok/Artifacts/osrb.md:21]: I got this back from ChatGPT when asking what the difference was:


> The Open Source Program Office (OSPO) and the Open Source Review Board (OSRB) are both crucial in managing an organization's engagement with open-source software, but they serve distinct functions and have different scopes of responsibility. Understanding the differences between them is key to appreciating how organizations can effectively leverage open-source software while mitigating risks and contributing back to the community.
> 
> ## Open Source Program Office (OSPO)
> An OSPO is a dedicated office or team within an organization that is responsible for coordinating and facilitating the organization's open-source activities. Its responsibilities are broad and strategic, encompassing the oversight of all open-source engagements, contributions, and compliance across the entire organization. Key functions of an OSPO include:
> 
> Strategy Development: Defining the organization's open-source strategy, including how open-source can be used to achieve business goals, and setting policies for open-source contributions and usage.
> Community Engagement: Building and maintaining relationships with the open-source community, including contributing to open-source projects and possibly launching open-source projects under the organization's banner.
> Education and Advocacy: Promoting the use of open-source within the organization, educating employees on best practices, and advocating for open-source values.
> Compliance and Governance: Ensuring that the organization complies with open-source licenses, managing intellectual property in open-source contributions, and setting up governance structures like the OSRB.
> Tooling and Infrastructure: Providing tools and infrastructure to support open-source development, contribution, and compliance within the organization.
> ## Open Source Review Board (OSRB)
> The OSRB, on the other hand, is typically a component of the governance structure that an OSPO might establish. It has a more focused role, primarily concerned with reviewing and approving open-source usage and contributions to ensure compliance with policies, licenses, and security standards. Key functions of an OSRB include:
> 
> Review and Approval: Evaluating proposed use of open-source components in projects and contributions to open-source projects to ensure they meet internal policies and license obligations.
> License Compliance: Ensuring that the organization's use of open-source software complies with the terms of open-source licenses, including obligations to disclose source code or attribute copyrights.
> Security Review: Assessing the security implications of using or contributing to open-source software and ensuring that security policies are adhered to.
> Policy Enforcement: Enforcing the organization's open-source policies at the project level and ensuring that developers and teams comply with these policies.
> ## Key Differences
> Scope and Focus: The OSPO has a broad, strategic focus, aiming to integrate open-source into the organization's overall strategy and operations. The OSRB has a more focused, operational role, concentrating on the review and approval process to ensure compliance and manage risks.
> Responsibilities: While the OSPO is responsible for the overarching open-source strategy, community engagement, and policy development, the OSRB is specifically tasked with the governance and oversight of open-source usage and contributions.
> Position in the Organization: An OSPO is a centralized office that oversees all open-source activities across the organization. An OSRB, however, is often a component of the governance structure established by the OSPO to handle specific tasks related to open-source compliance and security.
> In summary, while the OSPO sets the direction and policies for open-source engagement at a strategic level, the OSRB focuses on the governance and compliance aspects, ensuring that the organization's open-source activities align with internal policies and external license obligations.

**Review by robmoffat — COMMENTED** [review]
pretty good, although I think there should be talk of 'use' as well as 'contribution' in this

File: docs/bok/Artifacts/osrb.md (hunk 2 above)
Comment (robmoffat) [reply docs/bok/Artifacts/osrb.md:10]: apparently, I am wrong :)

File: docs/bok/Artifacts/osrb.md (hunk 4 above)
Comment (robmoffat) [reply docs/bok/Artifacts/osrb.md:21]: In discussion in today's OSR SIG, Amoel talked about how the OSRB provides a layer of governance and arbitration over the OSPO.  

His example:   should the firm use OpenTOFU?  It's a new open source fork of Terraform. Should they contribute? Or avoid?  This is something the OSRB would be expected to arbitrate on.

**Comment by robmoffat** [comment]
Some comments today from the OSR SIG:

NJ talked about the importance of the OSRB for enabling community engagement (which you discuss in the article).  The OSRB might be able to influence the incentive structure to allow for sponsorship of products, hiring, speaking, contributing, _gamification of desired behaviours_... all kinds of things.    

KX talked about how the OSPO is often trying to find a happy medium between competing internal forces such as compliance, comms etc.  An OSRB brings together these different forces in a structured way to bring clarity to what the OSPO should do.

AM gave a concrete example:  Bloomberg's OSRB identified projects that needed funding and then allocated funding (perhaps staff, perhaps financial incentives) to get those OSS projects maintained.

**Comment by robmoffat** [comment]
@BrittanyIstenes https://deploy-preview-256--open-source-readiness.netlify.app/docs/bok/Artifacts/osrb-rename

**Comment by robmoffat** [comment]
@BrittanyIstenes  https://deploy-preview-256--open-source-readiness.netlify.app/docs/bok/Artifacts/OSRB

File: docs/bok/Artifacts/OSRB.md [hunk 5]
```diff
@@ -1,16 +1,97 @@
 ---
 title: Open Source Review Board (OSRB)
 tags:
-  - Codebase Risk
-  - Dependency Risk
   - Staff Risk
-  - OSRB (Role)
+  - OPSO (Role)
```
Comment (psmulovics) [inline docs/bok/Artifacts/OSRB.md:5]: Should be OSPO

Comment (robmoffat) [reply docs/bok/Artifacts/OSRB.md]: good spot

**Review by psmulovics — APPROVED** [review]

**Pull request merged**